lib/sentry/client.ex

```diff
@@ -0,0 +1,42 @@
+defmodule Sentry.Client do
+  @sentry_version 5
+  quote do
+    unquote(@sentry_client "sentry-elixir/#{Mix.Project.config[:version]}")
+  end
+
+  def request(method, url, headers, body) do
```

Comment:
Since you can only post I'd probably just do a function that posts the body. I'd also have this function turn it into json.